### PR TITLE
chore: add param to update teams-app

### PR DESCRIPTION
**Pull request by nliu-ms**
non interactive mode, will continue update:
image

### CODE REVIEW

File: packages/fx-core/src/component/resource/appManifest/appStudio.ts [hunk 1]
```diff
@@ -747,31 +749,7 @@ export async function updateManifestV3(
...
-  const appStudioTokenRes = await ctx.tokenProvider.m365TokenProvider.getAccessToken({
-    scopes: AppStudioScopes,
-  });
-  if (appStudioTokenRes.isErr()) {
-    return err(appStudioTokenRes.error);
-  }
-  const appStudioToken = appStudioTokenRes.value;
-
   try {
-    const localUpdateTime = (await fs.stat(manifestFileName)).mtime.getTime();
```
Comment (nliu-ms) [inline packages/fx-core/src/component/resource/appManifest/appStudio.ts:759]: Remove this logic as file state timestamp cannot be used to distinguish local update time.

E.g.
User does not modify template, the local time will always be the same and less than dev portal update time.

Comment (HuihuiWu-Microsoft) [reply packages/fx-core/src/component/resource/appManifest/appStudio.ts:759]: In my understanding, localUpdateTime is always updated when building Teams app package. Correct me if I was wrong

**Review by HuihuiWu-Microsoft — DISMISSED** [review]

File: packages/cli/src/constants.ts [hunk 2]
```diff
@@ -87,6 +87,16 @@ export const AadManifestOptions: OptionsMap = {
   },
 };
 
+export const TeamsAppManifestFilePathName = "manifest-file-path";
+export const TeamsAppManifestOptions: OptionsMap = {
+  [AadManifestFilePathName]: {
```
Comment (Alive-Fish) [inline packages/cli/src/constants.ts:92]: Should it be `TeamsAppManifestFilePathName`?

File: packages/cli/src/utils.ts [hunk 3]
```diff
@@ -173,6 +173,20 @@ export async function askManifestFilePath(): Promise<Result<string, FxError>> {
   }
 }
 
+export async function askTeamsManifestFilePath(): Promise<Result<string, FxError>> {
+  const config: InputTextConfig = {
+    name: "manifest-file-path",
```
Comment (Alive-Fish) [inline packages/cli/src/utils.ts:178]: This can be `TeamsAppManifestFilePathName`.

**Review by Alive-Fish — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by 1yefuwang1 — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**